api/health/index.ts

```diff
@@ -1,34 +1,31 @@
-import * as healthcheck from '@hmcts/nodejs-healthcheck'
+import * as HealthCheck from '@hmcts/nodejs-healthcheck'
 import {SESSION, xuiNode} from '@hmcts/rpx-xui-node-lib'
-import {getConfigValue, showFeature} from '../configuration'
+import { getConfigValue, showFeature } from '../configuration'
 import {
   FEATURE_REDIS_ENABLED,
-  // FEATURE_TERMS_AND_CONDITIONS_ENABLED,
+  FEATURE_TERMS_AND_CONDITIONS_ENABLED,
   SERVICE_S2S_PATH,
+  SERVICES_CCD_COMPONENT_API_PATH,
+  SERVICES_CCD_DATA_STORE_API_PATH,
   SERVICES_DOCUMENTS_API_PATH,
   SERVICES_EM_ANNO_API_URL,
   SERVICES_IDAM_API_URL,
   SERVICES_IDAM_LOGIN_URL,
-  // SERVICES_TERMS_AND_CONDITIONS_URL
+  SERVICES_TERMS_AND_CONDITIONS_URL
 } from '../configuration/references'
 import * as log4jui from '../lib/log4jui'
 import {JUILogger} from '../lib/models'
-import {redisHealth} from './redis.health'
+const logger: JUILogger = log4jui.getLogger('RedisHealth')
```

Comment:
the logger name should be changed to be more generic as it's not only redis now

Reply:
understood